### ISSUE TITLE
Possible to bypass XSS filter with nodeNS

### ISSUE THREAD

**Issue by hpate-omicron** [elm/virtual-dom#168]
SSCCE: [Ellie](https://ellie-app.com/8qk75zTszH6a1)

```
module Main exposing (main)

import Html
import VirtualDom exposing (nodeNS)


main =
    nodeNS "http://www.w3.org/2000/svg"
        "script"
        []
        [ Html.text "alert('Hi')" ]
```

The user John J on Slack found it, he had this research. [Slack link](https://elmlang.slack.com/archives/C0CJ3SBBM/p1585166086062600)

The crux of it seems to be that `nodeNS` arguments are swapped between Elm and JS, on the elm side the tag is the first argument passed into `nodeNS`. [Link to code](https://github.com/elm/virtual-dom/blob/master/src/VirtualDom.elm#L71)

```
nodeNS : String -> String -> List (Attribute msg) -> List (Node msg) -> Node msg
nodeNS tag =
  Elm.Kernel.VirtualDom.nodeNS (Elm.Kernel.VirtualDom.noScript tag)
```

But on the JS side it is expecting the tag as the second argument
```
var _VirtualDom_nodeNS = F2(function(namespace, tag)
```

**Comment by lydell** [comment]
> The crux of it seems to be that `nodeNS` arguments are swapped between Elm and JS, on the elm side the tag is the first argument passed into `nodeNS`

This is close, but not quite what the issue is.

Compare `nodeNS`:

```elm
nodeNS : String -> String -> List (Attribute msg) -> List (Node msg) -> Node msg
nodeNS tag =
  Elm.Kernel.VirtualDom.nodeNS (Elm.Kernel.VirtualDom.noScript tag)
```

…with `keyedNodeNS`:

```elm
keyedNodeNS : String -> String -> List (Attribute msg) -> List ( String, Node msg ) -> Node msg
keyedNodeNS namespace tag =
  Elm.Kernel.VirtualDom.keyedNodeNS namespace (Elm.Kernel.VirtualDom.noScript tag)
```

`nodeNS` first parameter is _named_ wrong, and `noScript` is uselessly called on it. Here’s my suggestion:

```elm
nodeNS : String -> String -> List (Attribute msg) -> List (Node msg) -> Node msg
nodeNS namespace tag =
  Elm.Kernel.VirtualDom.nodeNS namespace (Elm.Kernel.VirtualDom.noScript tag)
```

**Comment by wclr** [comment]
Seems there is just just a little mistake made by @evancz.

**Comment by evancz** [comment]
Fixed in https://github.com/elm/virtual-dom/commit/9a389f84978930ca2f1dc2f84db505dafa018d3c which should be published shortly with version `1.0.3`. Thank you for the report, and thank you to @jfmengels for organizing the XSS issues and coordinating on testing.
